site/.vuepress/theme/layouts/Page.vue

```diff
@@ -4,10 +4,21 @@
     ref="content"
   >
     <Content itemprop="articleBody" />
+    <ContactForm v-if="path === 'contact'" />
```

Comment:
@labboy0276 this isnt the way we want to do it. just put the form directly on the `contact/index.md` page.

Reply:
@pirog I changed it to add the component in the markdown file.  I added the [name property](https://github.com/thinktandem/tandem/pull/190/files#diff-cbd523a603dd1183b56cfa91a91259feR198) to the component as well.  However I am seeing this message:

Unknown custom element: <ContactForm> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

Not sure what I am doing wrong

Reply:
@labboy0276 i think you need to add it as a global component

Reply:
@pirog yes, i fixed it and just pushed it up